index.html 

```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
+<head>
+	<title>Bubble Tea site</title>
+</head>
+<body>
+	<div class="topnav">
```

Comment:
Please ensure that ending and closing tags for the divs are all vertically aligned.

CORRECT:
<div>
</div>

INCORRECT:
<div>
   </div>

All <div>'s and their closing tags should all be vertically aligned.